handbook/engineering/developer-insights/code-insights/index.md

```diff
@@ -152,3 +152,13 @@ We use SVG and D3 libraries (d3-scales, d3-axis, d3-format) for this.
   If you want to dive a little more into the peculiarities of SVG spec.
 
 **NOTE**: _The order is not important, you can read these as you like depending on what you are interested in._
+
+## Code Insights history 
+
+In mid-2020, Code Insights was born out of [RFC 144: Code Insights](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit#), with the stated goal of being a Sourcegraph feature directly valuable to users in engineering leadership, in addition to individual developers. It was prototyped solely on the frontend to validate strong customer interest. 
+
+In late 2020, after the initial validation, and using the frontend-only functional prototype, the product team devoted time to create a detailed validation of use cases, customer excitement, potential directions, and the existing spaces the product fit into. Those findings are in [PD 18: Code Insights](https://docs.google.com/document/d/1d34gCpt_rUOMAun8phcjNsFofGaaA_N_8znmgaugdKw/edit). 
```

Comment:
It sounds like you're not giving yourself enough credit here, or was it really the whole product team vs you personally acting as the PM for our not-yet-formed team?

When I describe the history to candidates I actually always say "Mid last year we hired a PM who was excited about this feature too and did an amazing job at validating it with many of our customers, [...]". Basically, I see you joining the team as much of a milestone as our first engineers joining 🙂

Reply:
haha, it was just me + Alicja, I was just trying to avoid making the history about specific individuals since everyone should feel a part of the team. I'll play with the wording here to make it clear it was just 1 PM + 1 Designer

Reply:
```suggestion
In late 2020, after the initial validation, and using the frontend-only functional prototype, a product designer and newly-hired product manager developed a detailed validation of use cases, customer excitement, potential directions, and the existing spaces the product fit into. Those findings are in [PD 18: Code Insights](https://docs.google.com/document/d/1d34gCpt_rUOMAun8phcjNsFofGaaA_N_8znmgaugdKw/edit). 
```